app/src/main/java/com/fstyle/androidtrainning/MainActivity.java

```diff
@@ -2,12 +2,38 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
 
+    private Button mBtnMulti;
+    private EditText mEdtNumber1, mEdtNumber2;
+    private TextView mTxtResult;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        initViews();
+
+        mBtnMulti.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                float numb1 = Float.parseFloat(mEdtNumber1.getText().toString());
+                float numb2 = Float.parseFloat(mEdtNumber2.getText().toString());
+                float result = numb1 * numb2;
+                mTxtResult.setText("Result:   " + result);
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.